lib/advanced-open-file-view.coffee

```diff
@@ -411,6 +420,7 @@ class AdvancedFileView extends View
           suggestedPath = projectPaths[0] + path.sep
 
     @miniEditor.setText suggestedPath
+    @miniEditor.getModel().scrollToCursorPosition()
```

Comment:
Since `attach` is the only method that calls this, and since `attach` does this scrolling later on anyway, this is unnecessary.